spec/controllers/reauthn_required_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe ReauthnRequiredController do
+  let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
+
+  describe '#confirm_recently_authenticated' do
+    controller do
+      def show
+        render text: 'Hello'
+      end
+    end
+
+    before(:each) do
+      stub_sign_in(user)
+      routes.draw do
+        get 'show' => 'reauthn_required#show'
+      end
+    end
+
+    context 'recently authenticated' do
+      it 'allows action' do
+        get :show
+
+        expect(response.body).to eq 'Hello'
+      end
+    end
+
+    context 'authenticated outside the authn window' do
+      before do
+        controller.user_session[:authn_at] -= 60
```

Comment:
60 feels like a magic number here -- why did you pick this val? Perhaps putting it in a var would clarify